### PR TITLE
The better movement code with everything else intact.

### PR DESCRIPTION
**Pull request by GrimReaper2654**
This makes some of the old code redundant, but that isn't my problem anymore.

### CODE REVIEW

**Comment by GrimReaper2654** [comment]
This is literally the least amount of deletions I can make.

File: index.html [hunk 1]
```diff
@@ -43,6 +45,289 @@
...
 <body>
     <div id="canvasZone"><canvas id="renderCanvas"></canvas></div>
-    <script src="main.js"></script>
+    <script>
+        const sensitivity = 500; // larger is slower
+
+        var canvas = document.getElementById("renderCanvas");
+
+        var keyboard = {};
+        var vy = 0;
```
Comment (EddieTheEd) [inline index.html]: please comment what these variables mean...

**Comment by EddieTheEd** [comment]
looks good to me so far, i know you've fixed the whole mouse issue, please commit so i can look over again

**Comment by GrimReaper2654** [comment]
everything should be done.

**Comment by EddieTheEd** [comment]
will look over this soon!

**Comment by EddieTheEd** [comment]
Tom, please add a properly working pointer lock, and make the movement actually work (it jitters for some reason? this is because of the -= vy part of the code)

**Pull request merged**